manifest-list.md

```diff
@@ -36,7 +38,7 @@ For the media type(s) that this document is compatible with, see the [matrix][ma
 
   - **`platform`** *object*
 
-    This REQUIRED property describes the platform which the image in the manifest runs on.
+    This OPTIONAL property describes the platform which the image in the manifest runs on.
```

Comment:
Why do we make it as an optional property? Without this object, if a manifest list have multiple manifests, how can we know which platform (e.g., amd64, ppc64le) each manifest is for?